OlsztynHooligans/src/app/app.component.html

```diff
@@ -1,7 +1,10 @@
 <ul>
-    <li><a [routerLink]="['/login']">Home</a></li>
-    <li class="right"><a [routerLink]="['/register']">Register</a></li>
-    <li class="right"><a [routerLink]="['/login']">Login</a></li>
-    <li class="right"><a [routerLink]="['/login']" (click)="logout()">Logout</a></li>
+    <li><a [routerLink]="['/dashboard']">Home</a></li>
+    <li *ngIf="!isLogged" class="right"><a [routerLink]="['/register']">Register</a></li>
```

Comment:
Za dużo tego :/
Zrob tablice ktora ma miec mniej wiecej takie cos https://pastebin.com/3MeaaRrS
I druga ktora zawiera to co ma widziec zalogowany uzytkownik.

Nastepnie ngForem <li> przeleciec i uzupelnic te rzeczy danymi z ngFora.
W componentcie okreslasz co ma sie wyswietlic.